lib/rubycritic/generators/text/list.rb

```diff
@@ -0,0 +1,43 @@
+require "colorize"
+
+module Rubycritic
+  module Generator
+    module Text
+      class List
+        class << self
+          TEMPLATE_PATH = File.expand_path("../templates/list.erb", __FILE__)
+
+          def erb_template
+            @erb_template ||= ERB.new(File.read(TEMPLATE_PATH), nil, '-')
```

Comment:
Can you please use double quotes on this `-` String to make the RuboCop checks green?